src/lib/eina/eina_log.c

```diff
@@ -2189,7 +2187,7 @@ eina_log_print_cb_journald(const Eina_Log_Domain *d,
                                             "PRIORITY=%i", level,
                                             "MESSAGE=%s", message,
                                             "EFL_DOMAIN=%s", d->domain_str,
-                                            "THREAD=%lu", eina_thread_self_id(),
+                                            "THREAD=%lu", eina_thread_self(),
```

Comment:
Does it mean we're back to consider `self` can return the ID/integer type?

Reply:
Now a thread is always an `intptr_t`

Reply:
Oh ok.